utils/config/config_test.go

```diff
@@ -57,9 +57,9 @@ func TestConvertConfigV0ToV5(t *testing.T) {
 	defer cleanUpTempEnv()
 	content, err := convertIfNeeded([]byte(configV0))
 	assert.NoError(t, err)
-	configV5 := new(ConfigV5)
-	assert.NoError(t, json.Unmarshal(content, &configV5))
-	assertionV5Helper(t, configV5, coreutils.GetCliConfigVersion(), false)
+	configV6 := new(ConfigV6)
+	assert.NoError(t, json.Unmarshal(content, &configV6))
+	assertionHelper(t, configV6, 0, false)
 	assertCertsMigrationAndBackupCreation(t)
```

Comment:
Shouldn't the name fo this function be changed from `TestConvertConfigV0ToV5` to `TestConvertConfigV0ToLatest`?
The same question applies to `TestConvertConfigV1ToV5` and `TestConvertConfigV4ToV5`.